sdk/python/kfp/components/_python_op.py

```diff
@@ -769,98 +766,95 @@ def create_component_from_func(
     base_image: str = None,
     packages_to_install: List[str] = None,
 ):
-    '''
-    Converts a Python function to a component and returns a task factory (a function that accepts arguments and returns a task object).
-
-    Function name and docstring are used as component name and description.
-    Argument and return annotations are used as component input/output types.
-    Example::
-
-        def add(a: float, b: float) -> float:
-            """Returns sum of two arguments"""
-            return a + b
-
-        # add_op is a task factory function that creates a task object when given arguments
-        add_op = create_component_from_func(
-            func=add,
-            base_image='python:3.7', # Optional
-            output_component_file='add.component.yaml', # Optional
-            packages_to_install=['pandas==0.24'], # Optional
-        )
+    '''Converts a Python function to a component and returns a task factory (a function that accepts arguments and returns a task object).
 
-        # The component spec can be accessed through the .component_spec attribute:
-        add_op.component_spec.save('add.component.yaml')
+    Args:
+        func: The python function to convert
+        base_image: Optional. Specify a custom Docker container image to use in the component. For lightweight components, the image needs to have python 3.5+. Default is the python image corresponding to the current python environment.
+        output_component_file: Optional. Write a component definition to a local file. The produced component file can be loaded back by calling :code:`load_component_from_file` or :code:`load_component_from_uri`.
+        packages_to_install: Optional. List of [versioned] python packages to pip install before executing the user function.
 
-        # The component function can be called with arguments to create a task:
-        add_task = add_op(1, 3)
+    Returns:
+        A factory function with a strongly-typed signature taken from the python function.
+        Once called with the required arguments, the factory constructs a task instance that can run the original function in a container.
 
-        # The resulting task has output references, corresponding to the component outputs.
-        # When the function only has a single anonymous return value, the output name is "Output":
-        sum_output_ref = add_task.outputs['Output']
+    Examples:
+        The function name and docstring are used as component name and description. Argument and return annotations are used as component input/output types::
```

Comment:
Do we need extra indent here and below?

Reply:
Unfortunately yeah it won't display as the example in a code formatted box (see how it displays properly [here](https://al-kubeflow-pipelines.readthedocs.io/en/alexla-cleanup-sdk-docs/source/kfp.components.html#kfp.components.create_component_from_func)) without the indent.

![Screen Shot 2020-07-16 at 8 46 58 PM](https://user-images.githubusercontent.com/628146/87746548-b8ab8080-c7a5-11ea-86bc-70ccea66a333.png)